api_tests/base/test_sloan_study.py

```diff
@@ -269,3 +269,7 @@ def test_user_override_cookie_false(self, app, user, preprint):
         cookies = resp.headers.getall('Set-Cookie')
 
         assert f' dwf_{SLOAN_COI_DISPLAY}=False; Domain=.osf.io; Path=/; samesite=None; Secure' in cookies
+
+    def test_browseable_api(self, app):
+        resp = app.get('/v2/?format=api')
```

Comment:
Fun fact: https://api.osf.io/v2/?format=api works fine right now, while https://api.osf.io/v2/ 502s.

Reply:
It should be obvious seeing [the change](https://github.com/CenterForOpenScience/osf.io/pull/9375/files#diff-c1703080e33331cb3e1bfb4f1ddfeb01L193-R193) why that is, so I think this is still valid, but I'll throw in another test by content-type just to be safe.